src/test/java/com/qa/test/TC001.java

```diff
@@ -0,0 +1,13 @@
+package com.qa.test;
+
+import org.testng.annotations.Test;
+
+public class TC001 {
+
+	@Test
+	public void m1()
+	{
+		System.out.println("This is a newlt added Test Case");
```

Comment:
some gramatical mistake is there, fix it.

Reply:
ok I will do it